api/envoy/config/listener/v3/udp_listener_config.proto

```diff
@@ -33,8 +33,18 @@ message UdpListenerConfig {
 
   // Configuration for QUIC protocol. If empty, QUIC will not be enabled on this listener. Set
   // to the default object to enable QUIC without modifying any additional options.
-  // [#not-implemented-hide:]
-  // [#comment:Unhide when QUIC alpha is announced with other docs.]
+  //
+  // .. note::
+  //   QUIC support is currently alpha and should be used with caution.
+  //
+  //   For known outstanding issues before Envoy QUIC support is GA, you can
+  //   track https://github.com/envoyproxy/envoy/labels/quic-mvp for example
```

Comment:
I think that "For example" is the start of a new sentence? If so, perhaps a period and then Capitalize "for". If it's not the start of a new sentence, perhaps it can be rephrased for clarity?